packages/@react-aria/dnd/src/useDrag.ts

```diff
@@ -240,9 +241,11 @@ export function useDrag(options: DragOptions): DragResult {
   // If the dragged element is removed from the DOM via onDrop, onDragEnd won't fire: https://bugzilla.mozilla.org/show_bug.cgi?id=460801
   // In this case, we need to manually call onDragEnd on cleanup
 
-  useLayoutEffect(() => {
+  useEffect(() => {
```

Comment:
Switched to useEffect because layout effect cleanups happen before the element is actually removed.